governance-with-multisig/submit-upgrade.js

```diff
@@ -0,0 +1,47 @@
+const App = require('zos-lib').App;
+const Proxy = require('zos-lib').Proxy;
+const encodeCall = require('zos-lib').encodeCall;
+const fs = require('fs');
+const process = require('process');
+const MultiSigWallet = artifacts.require('MultiSigWallet')
+
+global.artifacts = artifacts;
+global.web3 = web3;
+
+async function submitUpgrade(networkName, contractName, multisigAddress) {
+  if (!contractName) {
+    throw Error("Contract name of the proxy to change ownership is required");
+  }
+  
+  const networkInfo = JSON.parse(fs.readFileSync(`zos.${networkName}.json`));
```

Comment:
you can use the `parseJson` from the `FileSystem` lib provided by `zos-lib` :)

Reply:
Yep, I wanted to avoid introducing new functions here, and just limit them to those specific to zOS